README.md

```diff
@@ -1 +1,99 @@
+| ovirt_engine_setup_db_name            | engine                | Engine database name. |
+| ovirt_engine_setup_db_user            | engine                | Engine database user. |
+| ovirt_engine_setup_db_password        | UNDEF                 | Engine database password. |
+
+* Engine Data Warehouse Database:
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_dwh_db_host        | localhost             | IP address or host name of a PostgreSQL server for DWH database. By default the DWH database will be configured on the same host as Engine. |
+| ovirt_engine_setup_dwh_db_port        | 5432                  | DWH database port. |
+| ovirt_engine_setup_dwh_db_name        | ovirt_engine_history  | DWH database name. |
+| ovirt_engine_setup_dwh_db_user        | ovirt_engine_history  | DWH database user. |
+| ovirt_engine_setup_dwh_db_password    | UNDEF                 | DWH database password. |
+
+* OVN related options:
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_provider_ovn_username | admin@internal     | Username for OVN. |
+| ovirt_engine_setup_provider_ovn_password | UNDEF              | Password for OVN. |
+
+* ISO domain related options: (This following options are deprecated for versions > 3.6)
+
+| Name                              | Default value         |  Description                               |
+|-----------------------------------|-----------------------|--------------------------------------------|
+| ovirt_engine_setup_configure_iso_domain | False                 | If `True`, ISO storage domain will be configured on Engine host.   |
+| ovirt_engine_setup_iso_domain_path      | /var/lib/exports/iso  | Path to a directory on Engine host to store data for ISO storage domain. |
+| ovirt_engine_setup_iso_domain_name      | IDO_DOMAIN            | Name of ISO domain.                         |
+| ovirt_engine_setup_iso_domain_acl       | 0.0.0.0/0.0.0.0(rw)   | ACL permissions for ISO domain mount point. |
+
+
+Dependencies
+------------
+
+None
+
+Example Playbook
+----------------
+
+```yaml
+---
+- hosts: engine
+  vars_files:
+    # Contains encrypted `ovirt_engine_setup_admin_password` variable using ansible-vault
+    - passwords.yml
+  vars:
+    ovirt_engine_setup_version: '4.1'
+    ovirt_engine_setup_organization: 'of.ovirt.engine.com'
+    ovirt_engine_setup_admin_password: "{{ ovirt_engine_setup_admin_password }}"
+    ovirt_repositories_ovirt_release_rpm: 'http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm'
```

Comment:
Please change to 4.2 (right now it does not exist, but it should exist before this roles is officially released)

Reply:
Done